tests/test_housing_price.py

```diff
@@ -0,0 +1,172 @@
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+from house_package_saachi import housing_price
+from sklearn.impute import SimpleImputer
+from sklearn.linear_model import LinearRegression
+from sklearn.metrics import mean_squared_error
+from sklearn.model_selection import StratifiedShuffleSplit, RandomizedSearchCV
+from sklearn.tree import DecisionTreeRegressor
+from sklearn.ensemble import RandomForestRegressor
+from scipy.stats import randint
+
+
+@pytest.fixture
+def housing_data():
+    housing_price.fetch_housing_data()
+    data = housing_price.load_housing_data()
+    return data
+
+
+def test_fetch_housing_data(tmpdir):
+    housing_path = tmpdir.mkdir("housing")
+    housing_url = housing_price.HOUSING_URL
+    housing_price.fetch_housing_data(housing_url, str(housing_path))
+    assert os.path.exists(os.path.join(str(housing_path), "housing.tgz"))
+
+
+def test_load_housing_data(housing_data):
+    assert not housing_data.empty
+
+
+def test_income_cat_proportions(housing_data):
+    housing_data["income_cat"] = pd.cut(
+        housing_data["median_income"],
+        bins=[0.0, 1.5, 3.0, 4.5, 6.0, np.inf],
+        labels=[1, 2, 3, 4, 5],
+    )
```

Comment:
@saachi-tiger in this assignment (**2.3**) we have asked only for installation testing

Reply:
Made changes to test_housing_price.py which test only the installation of package using pytest